.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "python.defaultInterpreterPath": "python",

```

Comment:
if you start using `venv` this should prob change to something like `"venv\\Scripts\\python.exe"`, but I don't remember the OS-agnostic way off-hand.